### PR TITLE
Add IOTA Sandbox docs

### LINKED ISSUE #1343: Create documentation for the IOTA Sandbox

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Added the IOTA Sandbox docs

## Links to any relevant issues

Fixes #1343.

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by Dr-Electron** [comment]
Preview: https://iota-wiki-i0e9lxwxi-iota1.vercel.app/iota-sandbox/welcome/

File: docs/build/iota-sandbox/docs/how-tos/setup-chain.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+To setup a chain you should have the wasp profile enabled. You can do this by uncommenting the `wasps` line in the `.env` file.
+
+Afterwards you can just follow the how-tos for [configuring wasp-cli](/wasp-cli/how-tos/wasp-cli/) and [setting up a chain](/wasp-cli/how-tos/setting-up-a-chain#start-the-chain).
+You can find the endpoints needed in the [endpoints section](./../references/endpoints.md).
+
+## Optional: Enable and configure EVM toolkit
+
+Enable the EVM toolkit by uncommenting the `evm-toolkit` line in the `.env` file.
+After that add change the `text` and `chainAddress` to your needs in the `assets/evm-toolkit/networks.json` file.
```
Comment (lucas-tortora) [inline docs/build/iota-sandbox/docs/how-tos/setup-chain.md:18]:
```suggestion
After that, change the `text` and `chainAddress` to your needs in the `assets/evm-toolkit/networks.json` file.
```

File: docs/build/iota-sandbox/docs/references/endpoints.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Chronicle
+
+API: `http://localhost/chronicle/`
+
+## EVM Toolkit
+
+:::info Profile
+
+EVM Toolkit is not active by default. You need to enable the `evm-toolkit` profile to use it.
```
Comment (lucas-tortora) [inline docs/build/iota-sandbox/docs/references/endpoints.md:45]:
```suggestion
The EVM Toolkit is not active by default. You need to enable the `evm-toolkit` profile to use it.
```

File: docs/build/iota-sandbox/docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+and [run](#run) the [IOTA Sandbox](welcome.md).
+It includes everything you need to develop your own IOTA dApps, plugins, and more.
+
+## Install the IOTA Sandbox using Docker
+
+### Requirements
+
+1. A recent release of [Docker](https://www.docker.com/). Avoid using the Docker version shipped with your OS since these are mostly out of date. You can find installation instructions in the [official Docker documentation](https://docs.docker.com/).
+2. [Docker Compose CLI plugin](https://docs.docker.com/compose/install/linux/).
+3. Make sure ports 80, 80... are not in use by other applications. If they are change them in the `.env` file.
```
[the remark this answers is not in the document]
Comment (Dr-Electron) [reply docs/build/iota-sandbox/docs/getting-started.md:16]: This was a placeholder 😓 I will update it with the correct ports 😆

File: docs/build/iota-sandbox/docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+### Download the latest release
+
+:::note 
+
+The commands assume you are using Linux.
+
+:::
```
[the remark this answers is not in the document]
Comment (Dr-Electron) [reply docs/build/iota-sandbox/docs/getting-started.md:24]: Yeah we use it in the hornet docs too

File: docs/build/iota-sandbox/docs/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+#### Set Up Your Environment
+
+In most cases the default setup should be enough. But you can edit the `.env` file to configure some optional settings:
+<Tabs groupId="network" queryString>
+<TabItem value="iota" label="IOTA">
+
+```sh reference
+https://github.com/dr-electron/iota-sandbox/blob/main/iota/.env
```
[the remark this answers is not in the document]
Comment (Dr-Electron) [reply docs/build/iota-sandbox/docs/getting-started.md:67]: Yes, once the aligned on a release schema for the sandbox 😆 
That's why this is still a draft

File: docs/build/iota-sandbox/docs/getting-started.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dr-Electron) [reply docs/build/iota-sandbox/docs/getting-started.md:16]:
```suggestion
3. Make sure ports 80, 8081 and 8082 are not in use by other applications. If they are change them in the `.env` file.
```

File: docs/build/iota-sandbox/docs/getting-started.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Dr-Electron) [reply docs/build/iota-sandbox/docs/getting-started.md:25]: Done

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**